server/api.go

```diff
@@ -26,6 +26,8 @@ func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Req
 	router.GET("/ai_threads", p.handleGetAIThreads)
 	router.GET("/ai_bots", p.handleGetAIBots)
 
+	router.POST("/conversation", p.handlePostConversation)
```

Comment:
I'm not overly worried about naming or scoping. Just started with this to get something working.